services/api/src/typeDefs.js

```diff
@@ -767,7 +767,7 @@ const typeDefs = gql`
     """
     Returns all Problems matching given filter (all if no filter defined)
     """
-    allProblems(source: [String], project: Int, environment: Int, envType: [EnvType], identifier: String, severity: [ProblemSeverityRating]): [Problem]
```

Comment:
This is technically a backwards compatibility break and would require a major version increase (candidate for lagoon 2?). Since lagoon is open source and the problems stuff is included in core, we should figure how to handle this. Maybe we can mark some part of the API as experimental to avoid holding stuff like this up? It's a similar issue with the billing stuff, were we are breaking stuff technically out of contract but it's ok for now because no one else is using the features.

More of a question/thought for @tobybellwood.